roc/version.go

```diff
@@ -0,0 +1,57 @@
+package roc
+
+/*
+#include <roc/version.h>
+*/
+import "C"
+
+import (
+	"strconv"
+	"strings"
+	"sync"
+)
+
+var (
+	versions    Versions
+	versionOnce sync.Once
+)
+
+const bindingsVersion = "0.2.0"
+
+// Version components.
+type Versions struct {
+	Library SemanticVersion // Native library version, libroc version.
+	Binding SemanticVersion // Go binding version.
+}
+
+// Semantic version components.
+type SemanticVersion struct {
+	Major uint64 // Major version component.
+	Minor uint64 // Minor version component.
+	Patch uint64 // Patch version component.
+}
+
+// Retrieve version numbers.
+// This function can be used to retrieve actual run-time version of the library.
+// It may be different from the compile-time version when using shared library.
+func Version() Versions {
+	versionOnce.Do(func() {
+		var cVersion C.struct_roc_version
+		C.roc_version_get(&cVersion)
```

Comment:
I think it's generally beneficial to check what do functions return, here for example some weird stuff can happen and C.roc_version_get can fail, then we'll just panic with no explanation. A more graceful shutdown stating that roc_version_get crapped out would be preferable.

Reply:
yup agree but seems from [API doc](https://roc-streaming.org/toolkit/docs/api/reference.html#roc-version) the roc_version_get will not return error, please CMIIW thanks

Reply:
I see, then I guess the API designers agreed on some null-like version (e. g. 0.0.0) that will indicate an error? @gavv would appreciate your input

Reply:
@Asalle it just can't fail, because it returns version hard-coded during build; thus no error code

Reply:
oh, I see, got it, then the implementation does makes sense. Good job @dhanusaputra!